### PR TITLE
Preventing empty messages by using existing notification data

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Quick fix for #258 

Simply use existing values when API data is out of sync.
Will do a follow up with an attempt of a more permanent and reliable fix.
But for now this will prevent the notifications with `0` values

# Testing

Unfortunately, it's not easy to reproduce, as we must rely on a race condition between the blockchain and the backend.
The best way to verify the fix is to keep placing orders and watch the console for this message: `computeFulfilledSummary::API data not yet in sync with blockchain`
When that happens, the notification would (before this fix) be `Swaped 0 token... etc`, and now you would get all values filled in.

### CODE REVIEW

File: src/custom/state/orders/updater.ts [hunk 1]
```diff
@@ -62,6 +62,8 @@ function _computeFulfilledSummary({
       // We only have the API order info, let's at least use that
       summary = `Swap ${sellToken} for ${buyToken}`
     }
+  } else {
+    console.log(`computeFulfilledSummary::API data not yet in sync with blockchain`)
```
Comment (anxolin) [inline src/custom/state/orders/updater.ts:66]: ubernit: If you really want to leave this console, I would give a name like `[state:orders:updater] computeFulfilledSummary`

**Review by anxolin — APPROVED** [review]
👌 good call. i still would go for the other solution

File: src/custom/state/orders/updater.ts [hunk 2]
```diff
@@ -46,7 +46,7 @@ function _computeFulfilledSummary({
 
   // if we can find the order from the API
   // and our specific order exists in our state, let's use that
-  if (orderFromApi) {
+  if (orderFromApi && Number(orderFromApi.executedBuyAmount) > 0 && Number(orderFromApi.executedSellAmount) > 0) {
```
[the remark this answers is not in the document]
Comment (alfetopito) [reply src/custom/state/orders/updater.ts:49]: I've extracted this into a function in the follow up PR.

File: src/custom/state/orders/updater.ts (hunk 1 above)
Comment (alfetopito) [reply src/custom/state/orders/updater.ts:66]: Sure, will update

**Pull request merged**